### DISCUSSION TITLE
Help with running group analyses in halfpipe

### DISCUSSION THREAD

**Discussion by EmeseK** [HALFpipe/HALFpipe#394] [Q&A] [answered]
Hi all,

I have preprocessed and ran my first-level analyses using halfpipe and after checking all the quality output I would like to run some group-level analyses now. However, I'm not sure how to proceed now:
- Is there a way to input the first-level output and only run the higher levels? Or do I always need to re-run the first-level analysis? or even re-run the preprocessing? 
- If not, would it be possible to input halfpipe first-level output into fsl directly?

Best, 

Emese

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @EmeseK,

- You can skip the first level processing and only run the higher level by adding the command line flag `--only-model-chunk` to the end of the command you use to start HALFpipe

- It is possible to input the results from HALFpipe into FSL directly. All the statistical maps you need are in the `derivatives/halfpipe` folder. Slide 15 of <https://osf.io/3hdex> shows some information about the file names of the statistical maps

Best,
Lea
